src/docs/markdown/caddyfile/directives/reverse_proxy.md

```diff
@@ -133,13 +133,18 @@ Caddy's proxy **transport** is pluggable:
 
 - **transport** defines how to communicate with the backend. Default is `http`.
 
+
 #### The `http` transport
 
 ```caddy-d
 transport http {
-	read_buffer  <size>
-	write_buffer <size>
-	dial_timeout <duration>
+	read_buffer             <size>
+	write_buffer            <size>
+    max_response_header     <size>
+	dial_timeout            <duration>
+    dial_fallback_delay     <duration>
+    response_header_timeout <duration>
+    expect_continue_timeout <duration>
```

Comment:
```suggestion
	read_buffer             <size>
	write_buffer            <size>
	max_response_header     <size>
	dial_timeout            <duration>
	dial_fallback_delay     <duration>
	response_header_timeout <duration>
	expect_continue_timeout <duration>
```

Those naughty spaces again

Reply:
Dangit sorry, we should add an `.editorconfig` in this repo tbh